code/modules/power/apc.dm

```diff
@@ -866,7 +858,7 @@
 		var/mob/living/silicon/ai/AI = user
 		var/mob/living/silicon/robot/robot = user
 		if(                                                             \
-			src.aidisabled ||                                            \
+			aidisabled ||                                            \
 			malfhack && istype(malfai) &&                                \
 			(                                                            \
 				(istype(AI) && (malfai!=AI && malfai != AI.parent)) ||   \
```

Comment:
Please single line this if statement. The multi lines and `\` make it even harder to read

Reply:
Disagree. It would be much harder to read if it was all on one line.
Yes, the \ are ugly, but they serve a purpose - they let you see which bits are ||ed or &&ed with which other bits.
If this was all on one line, parsing the logic would be much harder.